From 842737b191f76ce9929782d0c3658ded5ae238b4 Mon Sep 17 00:00:00 2001
From: Lukas Bezdicka <lbezdick@redhat.com>
Date: Tue, 16 Jun 2015 14:09:51 +0200
Subject: [PATCH] Fix support for puppet 4

We have to ensure config directory exists before creating file from template
---
 galera/manifests/server.pp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/galera/manifests/server.pp b/galera/manifests/server.pp
index eff45b1..5496ea2 100644
--- a/galera/manifests/server.pp
+++ b/galera/manifests/server.pp
@@ -142,6 +142,7 @@ class galera::server (
     group   => 'root',
     content => template('galera/wsrep.cnf.erb'),
     notify  => Service['mysqld'],
+    require => Class['mysql::server::install'],
   }
 
   if $manage_service {
